static/app/utils/analytics/insightAnalyticEvents.tsx

```diff
@@ -0,0 +1,67 @@
+export type InsightEventParameters = {
+  'insight.app_start.select_start_type': {type: string};
+  'insight.app_start.spans.filter_by_device_class': {filter: string};
+  'insight.app_start.spans.filter_by_operation': {filter: string};
+  'insight.app_start.spans.toggle_sample_type': {type: string};
+  'insight.asset.filter_by_blocking': {filter: string};
+  // Don't specify filter because page filter are arbitrary values
+  'insight.asset.filter_by_page': {};
+  'insight.asset.filter_by_type': {filter: string};
```

Comment:
Not, but this should probably match the module name, which is Assets (plural)

Reply:
thanks, will fix in the next